src/pages/install/windows.md

```diff
@@ -0,0 +1,91 @@
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Installing RGBDS on Windows
+
+The install instructions change a bit depending on the environment you wish to use RGBDS with.
+
+<Tabs>
+<TabItem value="wsl" label="WSL / Linux-like environments">
+
+For these, please refer to [the Linux instructions](linux) for the installed Linux distribution (the default Linux distribution on WSL is Ubuntu, whose package manager is `apt`).
+
+</TabItem>
+<TabItem value="cygwin" label="Cygwin / MSYS2">
+
+1. First, [pick the version you want to install](/docs).
+   If you want to [use `master`](/docs/master/#master) instead of a release, [follow the corresponding instructions](/install/master).
+2. Follow the "release page" link below "GitHub links", and grab either of the `win32` (for 32-bit Windows) or `win64` (for 64-bit Windows) `.zip` files, near the bottom of the page.
+3. Unzip that file, you should get the `.exe` files alongside a couple of `.dll`s.
+4. Copy all of those `.exe` and `.dll` files to the `/usr/local/bin` directory of Cygwin/MSYS2's installation.
+   (You can get its equivalent Windows path by running `cygpath -w /usr/local/bin`.)
+
+   **Do not put them in a subdirectory** (e.g. `/usr/local/bin/rgbds`)**!**
+   This would not work.
+
+After that, you should be able to use RGBDS from within the Cygwin/MSYS2 terminal, which you can confirm by running `rgbasm -V`.
+If this doesn't work, check that `/usr/local/bin` is within the PATH there (`echo $PATH`); if it isn't, you must add it (e.g. run `echo 'export PATH="/usr/local/bin:$PATH"' >>~/.bashrc`, and open a new Cygwin terminal).
+
+Note: if you can choose between using Cygwin or MSYS2, be advised that Cygwin is slower and has been reported to cause a bit of trouble to some.
+
+</TabItem>
+<TabItem value="win32" label="None of those">
+
+1. First, [pick the version you want to install](/docs).
+   If you want to [use `master`](/docs/master/#master) instead of a release, [follow the corresponding instructions](/install/master).
+2. Follow the "release page" link below "GitHub links", and grab either of the `win32` (for 32-bit Windows) or `win64` (for 64-bit Windows) `.zip` files, near the bottom of the page.
+3. Unzip that file, you should get the `.exe` files alongside a couple of `.dll`s.
+4. Either:
+   - ...put all of the files in a directory, then add it to the `PATH`.
+     This will **permanently** allow you to use RGBDS **from anywhere**.
```

Comment:
```suggestion
     This will *permanently* allow you to run RGBDS *from anywhere* in your file system.

     You can do this with the Windows GUI, the Command Prompt (cmd.exe), or PowerShell.
```

Reply:
Why "in your file system"?

Reply:
Probably so as not to confuse users, especially novices learning about `PATH` for the first time, into imagining remote desktop or mobile OS support.

Reply:
Then perhaps "from any directory" would work better?

Reply:
Or "from any folder", since this is Windows-oriented.